### PR TITLE
Remove encrypted variables support

### PR DESCRIPTION
**Pull request by walac**
The version we use of openpgp has security vulnerabilities, and it is
used only for the deprecated (and not used anymore) envrironment
variables feature.

So, we remove the encrypted variables support and the openpgp package.

### CODE REVIEW

**Comment by walac** [comment]
Good catch on the payload schema. If there is a task out there using this feature, it must be well hidden, I never saw a task using encrypted env. We can announce on tc mail list.

File: schemas/v1/payload.json [hunk 1]
```diff
@@ -160,15 +160,6 @@
...
     },
-    "encryptedEnv": {
-      "title": "List of encrypted environment variable mappings.",
-      "description": "List of base64 encoded asymmetric encrypted environment variables. See /docs/reference/workers/docker-worker/environment#encrypted-environment-variables",
-      "type": "array",
-      "items": {
-        "title": "Base64 encoded encrypted environment variable object.",
-        "type": "string"
-      }
-    },
```
Comment (djmitche) [inline schemas/v1/payload.json]: I forgot we don't have `additionalProperties: false` here, so this actually *won't* generate an error if the parameter is still used.  Oh well :(

Comment (walac) [reply schemas/v1/payload.json]: That comes from our old try on adding this restriction and breaking a bunch of ill-formed task payloads.

Comment (djmitche) [reply schemas/v1/payload.json]: Yup, I understand :)

At least with this change the variable won't be in the docs anymore.

**Pull request merged**